core-java-modules/core-java-exceptions-3/src/test/java/com/baeldung/exceptions/nosuchfielderror/NoSuchFieldErrorTest.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.exceptions.nosuchfielderror;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class NoSuchFieldErrorTest {

```

Comment:
Unit test class names should end in "UnitTest" as per guidelines. Otherwise, the PR will fail the build.